spec.html

```diff
@@ -6994,6 +6994,35 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-iteratorstepvalue" type="abstract operation">
+      <h1>
+        IteratorStepValue (
+          _iteratorRecord_: an Iterator Record,
+        ): either a normal completion containing either an ECMAScript language value or ~done~, or a throw completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It requests the next value from _iteratorRecord_.[[Iterator]] by calling _iteratorRecord_.[[NextMethod]] and returns either ~done~ indicating that the iterator has reached its end or the value from the IteratorResult object if a next value is available.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _result_ be Completion(IteratorNext(_iteratorRecord_)).
+        1. If _result_ is an abrupt completion, then
+          1. Set _iteratorRecord_.[[Done]] to *true*.
+          1. Return ? _result_.
+        1. Let _done_ be Completion(IteratorComplete(_result_)).
```

Comment:
`_result_` is a completion, but `IteratorComplete` wants an Object. You need to insert:
`Set _result_ to _result_.[[Value]].`

Reply:
or:
```suggestion
        1. Let _done_ be Completion(IteratorComplete(! _result_)).
```